package.json

```diff
@@ -18,14 +18,19 @@
     "Other"
   ],
   "activationEvents": [
-    "onCommand:extension.open"
+    "onCommand:extension.open",
+    "onCommand:extension.addQuickNote"
```

Comment:
"Add" seems a good keyword, What do you think of "Append" (like in IOS notes) or "Insert".

Reply:
Love it all, went with insert for now tell me how it looks